core/src/main/java/org/apache/accumulo/core/clientImpl/AccumuloClientImpl.java

```diff
@@ -298,82 +311,93 @@ private ClientInfo getClientInfo() {
     }
 
     @Override
-    public AccumuloClient build() throws AccumuloException, AccumuloSecurityException {
+    public T build() {
+      return builderFunction.build(this);
```

Comment:
If using `Function<ClientBuilderImpl, T>` for the build function type, instead of a custom interface, this method becomes `return builderFunction.apply(this);`

Reply:
Done